conf/routes

```diff
@@ -172,7 +172,7 @@ POST           /spaces/submit
 # EXTRACTORS
 # ----------------------------------------------------------------------
 GET            /extractors/:extractorName                                               @controllers.Extractors.showExtractorInfo(extractorName)
-
+GET            /extractorlog/:extractorName                                             @controllers.Extractors.showExtractorLog(extractorName)
```

Comment:
Can we change this to `/extractors:/extractorName/logs` to be more RESTful?

Reply:
yes